betterlockscreen

```diff
@@ -432,7 +432,11 @@ for arg in "$@"; do
 			if [[ ${2:0:1} = '-' ]]; then
 				shift 1
 			else
-				lockstyle="$2"; shift 2
+				if [[ -z "$2" ]]; then
+					source "$theme_rc"
```

Comment:
hmm, shouldn't this also `shift 1` ?

Reply:
I am unsure. I'm guessing the problem never arose in my installation since I never put more arguments.
If you think it should, I can quickly fix it up.

Reply:
please do

Reply:
Done :+1: